packages/math/package.json

```diff
@@ -39,6 +39,7 @@
   "dependencies": {
     "@keplr-wallet/types": "0.10.24-ibc.go.v7.hot.fix",
     "@keplr-wallet/unit": "0.10.24-ibc.go.v7.hot.fix",
-    "big-integer": "^1.6.48"
+    "big-integer": "^1.6.48",
+    "jest": "^29.6.1"
```

Comment:
I don't think we need this, it should be added as dev dependency in root package manifest.